Makefile

```diff
@@ -0,0 +1,55 @@
+WORKSPACE ?= VelocidiSDK.xcworkspace
+TEST_SDK ?= 14.4
+TEST_DEVICE ?= iPhone 11 Pro Max
+
+XCARGS := -workspace $(WORKSPACE) \
+					-sdk "iphonesimulator$(TEST_SDK)" \
+					-destination "platform=iOS Simulator,OS=$(TEST_SDK),name=$(TEST_DEVICE)" \
+					GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
+
+all: clean build
```

Comment:
Nitpick: I think that it is most common to have the default target be only the build phase. This allows one to take advantage of an incremental compilation while repeatedly calling `make`. 

`clean` is normally called separately.

Reply:
Yes, that makes more sense! Fixed in 173f363